Core/clim-core/dialog-views.lisp

```diff
@@ -70,6 +70,37 @@
                                          :default default
                                          :default-type default-type))
 
+(defun make-gadget-pane-from-view (view stream query-identifier &rest initargs)
+  (let* ((class-view (type-of view))
```

Comment:
Please use `class-of`

Reply:
Now I don't use `type-of` or `class-of` because the name of the gadget is stored in a class-allocated slot of the view instance.